content/docs/guides/branch-archiving.md

```diff
@@ -0,0 +1,145 @@
+</DocsList>
+
+</InfoBlock>
+
+To minimize storage costs, Neon automatically archives branches that are:
+
+- Older than **14 days**.
+- Have not been accessed for the past **24 hours**
+
+Both conditions must be true for a branch to be archived.
+
+## Unarchiving a branch
+
+**No action is required to unarchive a branch. It happens automatically.**
+
+Connecting to an archived branch, querying it, or performing some other action that accesses it will trigger the unarchive process. Connection times and query times may be slower while a branch is in the process of being unarchived.
+
+The following actions will automatically unarchive a branch, transferring the branch's data back to regular Neon storage:
+
+- [Connecting to or querying the branch from a client or application](/docs/connect/connect-from-any-app)
+- [Querying the branch from the Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor)
+- [Viewing the branch on the Tables page in the Neon Console](/docs/guides/tables)
+- [Creating a child branch](/docs/manage/branches#create-a-branch)
+- [Creating a role on a branch](/docs/manage/roles#create-a-role)
+- [Creating a database on a branch](/docs/manage/databases#create-a-database)
+- [Reset the branch from its parent](/docs/manage/branches#reset-a-branch-from-parent)
+- [Performing a restore operation on a branch](/docs/guides/branch-restore)
+- [Setting the branch as protected](/docs/guides/protected-branches)
+- Running [Neon CLI](/docs/reference/neon-cli) commands or [Neon API](https://api-docs.neon.tech/reference/getting-started-with-neon-api) calls that access the branch
+
+## Identifying archived branches
+
+Archived branches can be easily identified in the Neon Console. On the **Branches** page, archived branches are identified by an archive icon, as shown below:
+
+![the archive icon shown on a branch in the branches list page](/docs/guides/archived_branch_icon.png)
+
+If you select an archived branch on the **Branches** page to view its details, you can find out when the branch was archived:
+
+![the archive status shown on a branch in the branch detail page](/docs/guides/archived_branch_details.png)
+
+Archive and unarchive operations can also be monitored in the Neon Console or using the Neon API. See [Monitoring branch archiving](#monitoring-branch-archiving).
+
+## About archive storage
+
+For Neon projects created in AWS regions, inactive branches are archived in Amazon S3 storage. For Neon projects created in Azure regions, branches are archived in Azure Blob storage. For more information about how archive storage works in Neon, refer to [Archive storage](/docs/introduction/architecture-overview#archive-storage) in our architecture documentation.
+
+## Is automatic branch archiving configurable?
+
+Branch archiving is not configurable on the Neon Free Plan. When branch archiving becomes available on Neon paid plan, configuration options such as disabling archiving may be offered on those plans.
```

Comment:
Note: my current understanding is that this won't be directly exposed to users. It's more like "if you need this and you are on a paid plan contact sales". The current text reads like "you might have a toggle in UI for that"